app/routes.py

```diff
@@ -1 +1,354 @@
-from flask import Blueprint
\ No newline at end of file
+from app.models.task import *
+from app.models.goal import *
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import date
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+slack_token = os.environ.get("SLACK_TOKEN")
+
+# This is the BluePrint object that helps to "connect" my functions to
+# endpoints and HTTP methods for tasks.
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+#---------------post_task-----------
+# This function will be invoked when Flask receives an HTTP request with:
+#   method = POST
+#   endpoint = "/tasks"
+# This function creates a new Task instance with the information the client included
+# in the HTTP request. I get this information by using the global "request" variable.
+@tasks_bp.route("", methods=["POST"])
+def post_task():
+    # request.get_json() returns a dictionary with keys: "title", "description"
+    # Use corresponding values to create the Task instance.
+    post_dict = request.get_json()
+    print(post_dict)
+    if 'title' not in post_dict or 'description' not in post_dict:
+        return make_response({"details":"Invalid data"},400)
+    if 'completed_at' not in post_dict:
+        completed_at = None
+    else:
+        completed_at = post_dict['completed_at']
+    
+    title = post_dict['title']
+    description = post_dict['description']
+
+    task = Task(title=title, description=description, completed_at = completed_at)
```

Comment:
this works just fine! we can also combine these lines, so that we aren't making new variables

```suggestion
    task = Task(
        title=post_dict['title'],
        description=post_dict['description'],
        completed_at = completed_at
    )
```